src/configure/configure.ts

```diff
@@ -48,26 +64,38 @@ class PipelineConfigurer {
     private appServiceClient: AppServiceClient;
     private workspacePath: string;
     private uniqueResourceNameSuffix: string;
+    private telemetryHelper: TelemetryHelper;
 
-    public constructor() {
+    public constructor(telemetryHelper: TelemetryHelper) {
         this.inputs = new WizardInputs();
         this.inputs.azureSession = extensionVariables.azureAccountExtensionApi.sessions[0];
         this.azureDevOpsClient = new AzureDevOpsClient(this.inputs.azureSession.credentials);
         this.azureDevOpsHelper = new AzureDevOpsHelper(this.azureDevOpsClient);
         this.uniqueResourceNameSuffix = uuid().substr(0, 5);
+        this.telemetryHelper = telemetryHelper;
     }
 
     public async configure(node: any) {
+        this.telemetryHelper.setCurrentStep('GetAllRequiredInputs');
```

Comment:
Do want to go more granular here? for example, GetCodeInputs, GetPATToken, GetAzureDevOpsInputs, GetAzureInputs etc.

Reply:
I have created a new telemetry point : currentUserInput using which we can define the granularity and have added failure tracePoint to pint point the rest of the failures.